src/DevBetterWeb.Infrastructure/Handlers/DiscordLogMemberAddBookAddHandler.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Threading.Tasks;
+using DevBetterWeb.Core.Events;
+using DevBetterWeb.Core.Interfaces;
+using DevBetterWeb.Infrastructure.DiscordWebooks;
+
+namespace DevBetterWeb.Core.Handlers;
+
+public class DiscordLogMemberAddBookAddHandler : IHandle<MemberAddedBookAddEvent>
+{
+  private readonly BookDiscussionWebhook _webhook;
+
+  public DiscordLogMemberAddBookAddHandler(BookDiscussionWebhook webhook)
+  {
+    _webhook = webhook;
+  }
+
+  public static string returnWebhookMessageString(MemberAddedBookAddEvent domainEvent)
+  {
+    return $"User {domainEvent.Member.FirstName} {domainEvent.Member.LastName} just finished adding {domainEvent.Book.Title}! " +
```

Comment:
"User just added {title} to our book list!" instead of "just finished adding"

Reply:
changed in next PR.